addons/binding/org.openhab.binding.sinope/readme.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Binding for Sinopé devices
```

Comment:
Please change to "Sinopé Binding" - this is what our automatic documentation processing expects.